src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -44,93 +51,97 @@
   }
 
   /**
-   * Prompt the user for a valid password
+   * Prompt the user for a valid password.
+   * Password must not be empty or include spaces.
    *
-   * @return The password input
+   * @return password input.
```

Comment:
Returns `password input` is not as descriptive as it could be. I would suggest something like: `validated password` or `validated password entered by the user`.